### PR TITLE
Bump base64 dependency to 0.21.0

### PR DESCRIPTION
**Pull request by ema**

### CODE REVIEW

**Comment by tgonzalezorlandoarm** [comment]
Formatting error to fix :)

**Review by ionut-arm — APPROVED** [review]

File: src/key_info_managers/on_disk_manager/mod.rs [hunk 1]
```diff
@@ -252,7 +253,7 @@ fn key_triple_to_base64_filenames(key_triple: &KeyTriple) -> (String, String, St
 ///
 /// Returns an error as a string if either the decoding or the bytes conversion to UTF-8 failed.
 fn base64_data_to_string(base64_bytes: &[u8]) -> Result<String, String> {
-    match base64::decode_config(base64_bytes, base64::URL_SAFE) {
+    match base64::engine::general_purpose::URL_SAFE_NO_PAD.decode(base64_bytes) {
```
Comment (tgonzalezorlandoarm) [inline src/key_info_managers/on_disk_manager/mod.rs:256]: engine::general_purpose::URL_SAFE.decode(base64_bytes) ?

**Review by tgonzalezorlandoarm — CHANGES_REQUESTED** [review]

**Review by tgonzalezorlandoarm — DISMISSED** [review]
LGTM

**Comment by tgonzalezorlandoarm** [comment]
e2e_tests should have the neccesary changes as well :)

**Comment by tgonzalezorlandoarm** [comment]
`Cargo.lock` seems to be corrupt.
If you want to update base64 to version 0.21.0, my suggestion is to run:
```
git reset --soft HEAD^
git restore --staged Cargo.lock
cargo update --package base64@0.21.2 --precise 0.21.0
git add Cargo.lock
```
And commit again

**Review by tgonzalezorlandoarm — APPROVED** [review]

**Comment by tgonzalezorlandoarm** [comment]
LGTM Thanks!

**Pull request merged**